src/commands/heroku.rs

```diff
@@ -414,23 +415,76 @@ pub fn deploy_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandRe
 
     msg.reply(&ctx, build_response(&app_name, &build))?;
 
-    let mut build_pending = true;
+    let build_pending = Arc::new(Mutex::new(true));
```

Comment:
You can replace this with a [`std::sync::atomic::AtomicBool`](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicBool.html), which is better than a boolean in a mutex.

Reply:
Done!